### PR TITLE
[RFR] Use ExtractTextPlugin to deal with CSS

### LINKED ISSUE #55: Large black box across visualization
Issue received by email:

> When trying out the demo with the current CSS sheet (https://github.com/marmelab/EventDrops/blob/master/css/style.css) I get a large black box across the visualization.  If I pull down the CSS (http://marmelab.com/EventDrops/style.css) that is used for http://marmelab.com/EventDrops/ it renders correctly.

### PR DESCRIPTION
**Pull request by jpetitcolas**

### CODE REVIEW

**Comment by Kmaschta** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "css-loader": "0.19.0",
     "eslint": "1.6.0",
     "eslint-config-airbnb": "0.1.0",
+    "extract-text-webpack-plugin": "~1.0.1",
```
Comment (RobinBressan) [inline package.json]: You should fix the version like you did for others

**Comment by jpetitcolas** [comment]
Code review applied.

**Comment by Kmaschta** [comment]
But tests fail !

**Comment by RobinBressan** [comment]
Amazing it's green!

**Pull request merged**